app/serializers/generic_event_serializer.rb

```diff
@@ -5,17 +5,15 @@ class GenericEventSerializer
 
   set_type :events
 
-  attributes :occurred_at, :recorded_at, :notes
+  attributes :event_type, :occurred_at, :recorded_at, :notes
 
+  # TODO: should this/these be a belongs_to instead?
```

Comment:
Can you just change it without all the specs exploding?

Reply:
Done